### PR TITLE
Integrate ory editor for entities

### PR DESCRIPTION
**Pull request by Knorrke**
Note for checking this out: For this to work you need
- assets running 
- run the server (see PR in athene2-assets)
- change the ip adress to your local ip at `Markdown/Service/OryRenderService.php`. *(Note to myself: this has to be fixed before merge)*

### CODE REVIEW

File: src/module/Markdown/src/Markdown/Service/HtmlRenderService.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function render($input)
         $key = hash('sha512', $input);
 
         if ($this->storage->hasItem($key)) {
-            return $this->storage->getItem($key);
+            //return $this->storage->getItem($key);
```
Comment (Knorrke) [inline src/module/Markdown/src/Markdown/Service/HtmlRenderService.php]: fix this

File: src/module/Entity/src/Entity/Controller/PageController.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function indexAction()
         $this->layout('layout/3-col');
 
         if (!$entity->hasCurrentRevision()) {
-            $this->layout('layout/2-col');
+//            $this->layout('layout/2-col');
```
Comment (Knorrke) [inline src/module/Entity/src/Entity/Controller/PageController.php]: check

File: src/module/Markdown/src/Markdown/Service/OryRenderService.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        $key = hash('sha512', $input);
+
+        if ($this->storage->hasItem($key)) {
+            //return $this->storage->getItem($key);
+        }
+
+        $rendered = null;
+
+        $this->dnode->connect(
+            '192.168.176.112',
```
Comment (aeneasr) [inline src/module/Markdown/src/Markdown/Service/OryRenderService.php]: This must be changed before deployment

File: src/config/autoload/global.php [hunk 4]
```diff
@@ -38,35 +38,38 @@
...
         'redirect_strategy' => [
             'redirect_to_route_connected'    => 'authorization/forbidden',
             'redirect_to_route_disconnected' => 'authentication/login',
             'append_previous_uri'            => true,
-            'previous_uri_query_key'         => 'redir'
-        ]
+            'previous_uri_query_key'         => 'redir',
+        ],
+    ],
+    'assets_host'     => 'http://localhost:8081/',
```
Comment (Knorrke) [inline src/config/autoload/global.php]: ping^^

**Pull request merged**